OpenRA.Game/Actor.cs

```diff
@@ -22,7 +22,7 @@
 
 namespace OpenRA
 {
-	public sealed class Actor : IScriptBindable, IScriptNotifyBind, ILuaTableBinding, ILuaEqualityBinding, ILuaToStringBinding, IEquatable<Actor>, IDisposable
+	public sealed class Actor : IScriptBindable, IScriptNotifyBind, ILuaTableBinding, ILuaEqualityBinding, ILuaToStringBinding, ISync, IEquatable<Actor>, IDisposable
```

Comment:
There is a [Sync] attribute on the player through, which seems like a reasonable thing to check. I'm kinda leaving it around as a mini-reminder to fix it eventually so we sync the actor correctly but until then I need ISync here to make the lint pass for now.

I could remove it if preferred.